src/personality.coffee

```diff
@@ -0,0 +1,104 @@
+# These classes are a prime candidate to become their own module, but not now
+_ = require 'lodash'
+
+class Personality
+	###*
+	* Creates a Personality object that handles creating flavourful messages
+	* for a variety of circumstances
+	* @param {string} personality - index of the personality to load
+	###
+	constructor: (personality) ->
+		personalities =
+			rude:
+				confirm: ['Done it.']
+				holding: ["Shut up and wait, I'll do it"]
+				departure: ['You can go away now.']
+				deny: ["Sod off, I'm not doing that"]
```

Comment:
Can we get consistent quotes on these strings please? The standard says single quotes `'` for non interpolation strings, but at the very least these should all be the same.